incident_test.go

```diff
@@ -747,3 +747,51 @@ func TestIncident_ManageAlerts(t *testing.T) {
 	}
 	testEqual(t, want, res)
 }
+
+// CreateIncidentStatusUpdate
+func TestIncident_CreateIncidentStatusUpdate(t *testing.T) {
```

Comment:
This test does not appear to assert that the request was formatted as expected before sending it to the API (HTTP endpoint). Could you expand this test to actually assert that the request looks correct for both the `From` header and the `message` in the request body?

Maybe this other test can be of inspiration, with some changes being needed to work for this specific request: https://github.com/PagerDuty/go-pagerduty/blob/9190a3b061127b6b934a72c442843a3ee51bec61/incident_test.go#L54-L81

Reply:
Thanks. Done.